recipes/rlp/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
It looks like the `LICENSE` is already included in the tarball, so you don't need to add it here. You should be able to leave the `license_file` below unchanged, however.

Reply:
I thought that I still have to add it, to be safe. But ok.

Reply:
No, because the build will error out if it can't find the file specified by `license_file`.